extensions/replication/tasks/ReplicateObject.js

```diff
@@ -34,10 +34,14 @@ class ReplicateObject extends BackbeatTask {
     constructor(qp) {
         const qpState = qp.getStateVars();
         super({
-            retryTimeoutS: qpState.repConfig.queueProcessor.retryTimeoutS,
+            retry: qpState.repConfig.queueProcessor.retry,
         });
         Object.assign(this, qpState);
-
+        this.destType = this.destConfig.bootstrapList
+        .find(endpoint => endpoint.site === this.site);
+        const retryParams = this.repConfig.queueProcessor.retry[this.destType];
```

Comment:
We'll have to have a way of accounting for a bootstrap list type of `aws_s3` and the newly added retry config property `aws`.

Reply:
Yeah it should be called `aws_s3`. I need to find a way to have one source of truth for types (refactor for later).